internal/deb/extract.go

```diff
@@ -289,3 +428,12 @@ func parentDirs(path string) []string {
 	}
 	return parents
 }
+
+// sanitizeTarPath removes the leading "./" from the source path in the tarball,
+// and verifies that the path is not empty.
+func sanitizeTarPath(path string) string {
```

Comment:
Do we have any logic that prevents the tar from leaving the target directory?

Reply:
No, we don't at the moment.

Reply:
Agreed offline to do it, just not in this PR.